vignettes/troubleshooting.Rmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "Troubleshooting"
+author: "KimYen Ladia"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Troubleshooting}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+## Troubleshooting
+
+For users who are using a Windows machine and a network mapped drive, where your folder mapped from a drive `H:\\Documents\\R\\win-library\\3.4` to a network location `\\\\samba-xxx/home/<your_home_dir>/Documents/R/win-library/3.4`, you may experience the following error:
+```{r eval = F}
+> library(synapser)
+
+Loading required package: PythonEmbedInR
+
+Error: package or namespace load failed for ‘PythonEmbedInR’:
+
+.onLoad failed in loadNamespace() for 'PythonEmbedInR', details:
+
+  call: library.dynam.unload("PythonEmbedInR", packageRootDir)
+
+  error: DLL ‘PythonEmbedInR.dll’ was not loaded
+
+Error: package ‘PythonEmbedInR’ could not be loaded
+```
+
+First, verify that in R, `.libPaths()` points to one location:
```

Comment:
First, verify that is R, .libPaths() uses the non-Windows reference to the network file share, for example: